Mathlib/LinearAlgebra/Eigenspace/Triangularizable.lean

```diff
@@ -13,29 +13,35 @@ import Mathlib.FieldTheory.IsAlgClosed.Spectrum
 
 This file contains basic results relevant to the triangularizability of linear endomorphisms.
 
-* Every linear operator on a vector space over an algebraically closed field has an eigenvalue.
-* The generalized eigenvectors span the entire vector space.
+## Main definitions / results
+
+ * `Module.End.iSup_generalizedEigenspace_eq_top`: in finite dimensions, over an algebraically
+   closed field, the generalized eigenspaces of any linear endomorphism span the whole space.
+ * `Module.End.iSup_generalizedEigenspace_restrict_eq_top`: in finite dimensions, if the
+   generalized eigenspaces of a linear endomorphism span the whole space then the same is true of
+   its restriction to any invariant submodule.
```

Comment:
Please add these descriptions as docstrings to these results. I understand the names after staring at the type for 5 secs. I guess these natural language docstrings might speed up that process.

Reply:
Good point; done.